<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefsChat</class>
 <widget class="QWidget" name="PrefsChat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_title">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Chat</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="_chat_name">
     <item row="0" column="0">
      <widget class="QLabel" name="ChatName_label">
       <property name="text">
        <string>Our chat &amp;name:</string>
       </property>
       <property name="buddy">
        <cstring>ChatName</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="ChatName"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="AutoAllowSnooping">
     <property name="text">
      <string>Automatically allow &amp;snoop</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="_incoming_calls">
     <property name="title">
      <string>Incoming calls</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="_incoming">
        <item>
         <widget class="QCheckBox" name="AcceptChatConnections">
          <property name="text">
           <string>&amp;Accept incoming calls on port:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="ChatPort">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="ValidateIncomingChatCalls">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&amp;Validate caller</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="_message_appearance">
     <property name="title">
      <string>Message Appearance</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="_color">
        <item>
         <widget class="QLabel" name="ChatColorsForeground_label">
          <property name="text">
           <string>Colour:</string>
          </property>
          <property name="buddy">
           <cstring>ChatColorsForeground</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ColorPickerButton" name="ChatColorsForeground">
          <property name="text">
           <string/>
          </property>
          <property name="autoRaise" stdset="0">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ChatColorsBackground_label">
          <property name="text">
           <string>on</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ColorPickerButton" name="ChatColorsBackground">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="IgnoreChatColors">
        <property name="text">
         <string>&amp;Ignore incoming colours</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="_prefix">
        <item row="0" column="0">
         <widget class="QLabel" name="ChatMessagePrefix_label">
          <property name="text">
           <string>&amp;Prefix messages:</string>
          </property>
          <property name="buddy">
           <cstring>ChatMessagePrefix</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="ChatMessagePrefix"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="_size_limits">
     <property name="title">
      <string>Message size limits</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="ChatMaxLinesPerMessage_label">
        <property name="text">
         <string>Limit lines per message:</string>
        </property>
        <property name="buddy">
         <cstring>ChatMaxLinesPerMessage</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="ChatMaxLinesPerMessage">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="specialValueText">
         <string>∞</string>
        </property>
        <property name="maximum">
         <number>1000000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="ChatMaxBytesPerMessage_label">
        <property name="text">
         <string>Limit bytes per message:</string>
        </property>
        <property name="buddy">
         <cstring>ChatMaxBytesPerMessage</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="ChatMaxBytesPerMessage">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="specialValueText">
         <string>∞</string>
        </property>
        <property name="maximum">
         <number>1000000</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="_incoming_files">
     <property name="title">
      <string>Incoming Files</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QCheckBox" name="AutoAllowFiles">
        <property name="text">
         <string>Automatically accept &amp;files</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="_save_to">
        <item>
         <widget class="QLabel" name="ChatFileSaveDirectory_label">
          <property name="text">
           <string>Save &amp;to:</string>
          </property>
          <property name="buddy">
           <cstring>ChatFileSaveDirectory</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ChatFileSaveDirectory"/>
        </item>
        <item>
         <widget class="QToolButton" name="ChatFileSaveDirectory_browse">
          <property name="text">
           <string>&amp;Browse...</string>
          </property>
          <property name="icon">
           <iconset theme="document-open"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="spacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorPickerButton</class>
   <extends>QPushButton</extends>
   <header>colorpickerbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>AcceptChatConnections</sender>
   <signal>toggled(bool)</signal>
   <receiver>ChatPort</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AcceptChatConnections</sender>
   <signal>toggled(bool)</signal>
   <receiver>ValidateIncomingChatCalls</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
